lib/common_test/doc/src/write_test_chapter.xml

```diff
@@ -283,9 +283,9 @@
     <c>proplists:get_value</c>). Example: <c>PrivDir = ?config(priv_dir, Config)</c>.
     </p>
 
-    <p>If the test case function crashes or exits purposely, it is considered
-    <em>failed</em>. If it returns a value (no matter what value), it is
-    considered successful. An exception to this rule is the return value
+    <p>If the test case function crashes, exits purposely or returns <c>{fail,Reason}</c>,
+    it is considered <em>failed</em>. If it returns any other value (no matter what value),
+    it is considered successful. An exception to this rule is the return value
```

Comment:
As we are improving this I was thinking maybe we should rephrase to 

"If {skip, Reason}  is returned the test case will be skipped and the Reason logged, otherwise if it returns any other value (no matter what value), the test case is considered successful."  I think it sounds better!

Reply:
I think return value of a testcase is an important part of this section and possibly being common reason people visiting this chapter. IMHO having single paragraph for describing all possibilities is not optimal here and not easy to find when not reading sequentially.

Maybe we could use some form of enumeration to make it more visible what return value could be and how it will be interpreted (similarly to how it is explained in previous init/end_per_testcase section)?